File: android/src/main/java/org/ligi/gobandroid_hd/ui/fragments/CommentAndNowPlayingFragment.java
package org.ligi.gobandroid_hd.ui.fragments;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import org.ligi.gobandroid_hd.R;
import org.ligi.gobandroid_hd.ui.go_terminology.GoTerminologyViewActivity;

public class CommentAndNowPlayingFragment extends GobandroidGameAwareFragment {

    private TextView myTextView;

    @Override
    public View createView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        final View res = inflater.inflate(R.layout.game_extra_review, container, false);
        myTextView = (TextView) res.findViewById(R.id.comments_textview);
        onGoGameChange();
        return res;
    }

    @Override
    public void onGoGameChange() {
        getActivity().runOnUiThread(new Runnable() {

            @Override
            public void run() {
                if (myTextView != null) {
                    myTextView.setText(game.actMove.getComment());
                    GoTerminologyViewActivity.Companion.linkifyTextView(myTextView);
                }
            }

        });
    }

}

File: android/src/main/java/org/ligi/gobandroid_hd/ui/fragments/CommentAndNowPlayingFragment.kt
package org.ligi.gobandroid_hd.ui.fragments

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import kotlinx.android.synthetic.main.comment_textview.*
import org.ligi.gobandroid_hd.R
import org.ligi.gobandroid_hd.events.GameChangedEvent
import org.ligi.gobandroid_hd.ui.go_terminology.GoTerminologyViewActivity

class CommentAndNowPlayingFragment : GobandroidGameAwareFragment() {

    override fun createView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        val res = inflater.inflate(R.layout.game_extra_review, container, false)
        onGoGameChanged(null)
        return res
    }

    override fun onGoGameChanged(gameChangedEvent: GameChangedEvent?) {
        super.onGoGameChanged(gameChangedEvent)
        activity.runOnUiThread {
            comments_textview?.let {
                it.text = game.actMove.comment
                GoTerminologyViewActivity.linkifyTextView(it)
            }
        }
    }

}
